### ISSUE TITLE
ConvNeXt `classifier_activation=softmax` returns linear layer

### ISSUE THREAD

**Issue by Frightera** [keras-team/keras#17386]
**Describe the problem**.

Specifying `classifier_activation=softmax` in ConvNeXt models does not work, instead it returns the model with a linear classifier layer.

Inspecting source code:
https://github.com/keras-team/keras/blob/e6784e4302c7b8cd116b74a784f4b78d60e83c26/keras/applications/convnext.py#L524-L526

Definition of `Head` function:
https://github.com/keras-team/keras/blob/e6784e4302c7b8cd116b74a784f4b78d60e83c26/keras/applications/convnext.py#L327-L348

Find the Colab [gist here](https://colab.research.google.com/gist/Frightera/31d1c09bc6c882c35f15afa1310abb6b/convnext-classifier_activation.ipynb).

If OK I can open a PR and fix it.

**Comment by innat** [comment]
FYI, all models are moving to keras-cv. https://github.com/keras-team/keras-cv/pull/678

**Comment by Frightera** [comment]
I see, in keras-cv the implementation is correct. Shall I close this issue?

**Comment by innat** [comment]
Yes, you may. All the models from application will be moved to kcv completely soon.

**Comment by innat** [comment]
@Frightera please re-open this ticket. 

> Shall I close this issue?
> yes.

I was wrong. This should be discussed and fixed right away.

**Comment by Frightera** [comment]
@innat Sure, re-opened it. Should be a small and easy fix.

**Comment by Frightera** [comment]
Closing as PR was merged.
